### PR TITLE
Sort Students in order in award exp page

### LINKED ISSUE #306: Ordering of Students in Award EXP (Admin Function)
Currently the ordering of Students when in the Award EXP page is not by Alphabetical order. 

To provide the option to do allow for this kind of sorting.

### PR DESCRIPTION
**Pull request by weiqingtoh**
Fixes #306 Sorts Students in order to allow manual awarding of EXP to be slightly easier.

### CODE REVIEW

**Comment by weiqingtoh** [comment]
@jsyeo please review and merge, thanks!

File: app/controllers/manual_rewards_controller.rb [hunk 1]
```diff
@@ -14,6 +14,9 @@ def manual_exp
       end
     end
 
+    # sort students by name, capitalize to make sorting indiferent to case of names
+    @student_courses.sort_by! { |student| student.name.capitalize }
```
Comment (jsyeo) [inline app/controllers/manual_rewards_controller.rb]: Should sort in model actually.

Comment (jsyeo) [inline app/controllers/manual_rewards_controller.rb]: And use order, http://apidock.com/rails/ActiveRecord/QueryMethods/order

**Comment by weiqingtoh** [comment]
@jsyeo Updated - pls review and merge! Thanks!

**Comment by jsyeo** [comment]
:+1:

**Pull request merged**